### PR TITLE
Replace astro.h defines with constexpr

### PR DESCRIPTION
**Pull request by ajtribick**

### CODE REVIEW

File: src/celengine/star.cpp [hunk 1]
```diff
@@ -985,7 +985,7 @@ float Star::getRadius() const
 
     // Use the Stefan-Boltzmann law to estimate the radius of a
     // star from surface temperature and luminosity
-    return SOLAR_RADIUS * (float) sqrt(lum) *
+    return SOLAR_RADIUS<float> * (float) sqrt(lum) *
```
Comment (375gnu) [inline src/celengine/star.cpp:988]: `(float)` is not needed

File: src/celephem/customorbit.cpp [hunk 2]
```diff
@@ -569,7 +569,7 @@ class EarthOrbit : public CachingOrbit
 
         eclLong = nu+degToRad(ls-ms+dl) + celestia::numbers::pi;
         eclLong = pfmod(eclLong, TWOPI);
-        distance = KM_PER_AU * (1.0000002*(1-s*cos(ea))+dr);
+        distance = KM_PER_AU<double> * (1.0000002*(1-s*cos(ea))+dr);
```
Comment (375gnu) [inline src/celephem/customorbit.cpp:572]: `1` -> `1.0`

File: src/celengine/parseobject.cpp [hunk 3]
```diff
@@ -75,7 +75,7 @@ GetDefaultUnits(bool usePlanetUnits, double& distanceScale, double& timeScale)
 static void
 GetDefaultUnits(bool usePlanetUnits, double& distanceScale)
 {
-    distanceScale = (usePlanetUnits) ? KM_PER_AU : 1.0;
+    distanceScale = (usePlanetUnits) ? KM_PER_AU<double> : 1.0;
```
Comment (375gnu) [inline src/celengine/parseobject.cpp:78]: `()` around `usePlanetUnits` are not needed

**Review by 375gnu — CHANGES_REQUESTED** [review]

File: src/celengine/parseobject.cpp (hunk 3 above)
Comment (ajtribick) [reply src/celengine/parseobject.cpp]: done

File: src/celephem/customorbit.cpp (hunk 2 above)
Comment (ajtribick) [reply src/celephem/customorbit.cpp]: done

File: src/celengine/star.cpp (hunk 1 above)
Comment (ajtribick) [reply src/celengine/star.cpp]: done

**Review by 375gnu — APPROVED** [review]

**Pull request merged**